docs/blog/november-contributions-showcase.md

```diff
@@ -0,0 +1,137 @@
+
+You can see the source [here](https://github.com/fuzzyma/e18e-tools) and an example of the produced output [here](https://github.com/es-tooling/ecosystem-cleanup/issues/137#issue-2702026306).
+
+This is such a useful tool since we have no easy API access to find this information right now.
+
+In many of the issues of the [cleanup project](https://github.com/es-tooling/ecosystem-cleanup/), we now have an incredibly useful list of all the top dependents of the target package. This is making the cleanup work so much easier.
+
+Big thanks to [@fuzzyma](https://bsky.app/profile/fuzzyma.bsky.social) and [@devminer.xyz](https://bsky.app/profile/devminer.xyz) for making this a thing!
+
+## Contributions / Improvements
+
+### bluesky
+
+Some fun collaboration kicked off in the [e18e discord](https://chat.e18e.dev) recently. A few members of the community started looking into the bluesky bundle size, and noticed a few inefficiencies :eyes:
+
+While digging around, they noticed [zod](https://github.com/colinhacks/zod) was included twice in the bundle!
+
+A simple lockfile update allowed this to be de-duped, and shaved off a fair chunk of the bundle.
+
+Super work here by [@marvinh.dev](https://bsky.app/profile/marvinh.dev) and [@jviide.iki.fi](https://bsky.app/profile/jviide.iki.fi) :tada:
+
+You can read more about the investigation and the solution in the [pull request](https://github.com/bluesky-social/social-app/pull/6610).
+
+## Other news
+
+### deoptexplorer
+
+More of a "you should know about this" than news - [deoptexplorer](https://github.com/microsoft/deoptexplorer-vscode) is an extremely useful VSCode plugin to help you find deoptimisations in your code.
+
+In the community, we have started using it heavily to find performance bottlenecks in many different popular tools.
+
+Big thanks to microsoft for building this! Check it out and maybe you'll find some performance improvements :rocket:
+
+### `require(esm)` unflagged in Node 22.x
+
+In Node 22.x, the ability to `require(...)` ES modules has been unflagged and enabled by default!
+
+This is huge news and should open many doors for us being able to migrate libraries to ESM-only. We should also be able to gradually start removing dual-package publishing from various popular packages and go all in on ESM.
+
+Read more about this in [Node's blog post](https://nodejs.org/en/blog/release/v22.12.0#requireesm-is-now-enabled-by-default).
+
+Excellent work by [@joyeecheung](https://bsky.app/profile/joyeecheung.bsky.social) as always :pray:
+
+### vite 6.0
+
+A big milestone this month - vite 6.0 was released!
+
+This new release has a huge number of changes and, in the context of e18e, has a great list of performance improvements many of our community worked on.
+
+Two big changes are the upgrade of [chokidar](https://github.com/paulmillr/chokidar) to 4.x, and the migration to [tinyglobby](https://github.com/SuperchupuDev/tinyglobby). These two alone reduced the bundle size greatly :broom:
```

Comment:
Sorry for the late review. I just noticed this part but Vite 6 had reverted to using chokidar v3 still.